src/plan/global.rs

```diff
@@ -777,11 +797,11 @@ impl<VM: VMBinding> BasePlan<VM> {
         }
 
         debug!(
-            "self.get_pages_reserved()={}, self.get_total_pages()={}",
-            plan.get_pages_reserved(),
+            "self.get_reserved_pages()={}, self.get_total_pages()={}",
+            plan.get_reserved_pages(),
             plan.get_total_pages()
         );
-        let heap_full = plan.get_pages_reserved() > plan.get_total_pages();
+        let heap_full = plan.get_reserved_pages() > plan.get_total_pages();
```

Comment:
Maybe add a small comment here just to say that reserved pages > total pages in case of OOM and we have a copying collector?

Reply:
Sorry I don't get what you meant.

Reply:
I meant that this case can only happen when we have an OOM right? I thought we should document it here for the future.

Reply:
I am a bit confused when you said OOM. OOM means we run out of memory and we will tell the binding that we are out of memory, right? This is not OOM. This is just a check for GC, each slowpath allocation will go through this check.

Reply:
Right. Sorry I should have clarified. I am referring to this comment: https://github.com/mmtk/mmtk-core/pull/535#discussion_r796244445

Reply:
Ah sorry, my bad 😅